planet/scripts/v1.py

```diff
@@ -405,28 +405,37 @@ def list_feeds(pretty, limit, stats):
     echo_json_response(response, pretty, limit)
 
 
-@feeds.command('list-mosaics')
-@click.argument('feed_id')
-def get_mosaic_list_for_feed(feed_id):
-    '''List mosaics linked to feed'''
-    cl = clientv1()
-    feed_info = cl.get_feed_info(feed_id).get()
+def print_feed_mosaic_list(feed_info, client):
+    """Print the listing of mosaics for a feed."""
```

Comment:
I factored this function out of three functions in the module. "Rule of 3" argues for this factor, plus I'm also fixing 3 * 2 bugs.

Reply:
whats the rule of 3?

Reply:
https://wiki.c2.com/?ThreeStrikesAndYouRefactor

Reply:
I like this take, too https://andrewbrookins.com/technology/the-rule-of-three/

> The principle behind the rule is that it’s easier to make a good abstraction from duplicated code than it is to refactor the wrong abstraction. That’s my take, anyway.